app/inbound/save-invoice-lines.js

```diff
@@ -0,0 +1,12 @@
+const db = require('../data')
+
+const saveInvoiceLines = async (invoiceLines, paymentRequestId, transaction) => {
+  for (const invoiceLine of invoiceLines) {
+    delete invoiceLine.invoiceLineId
+    invoiceLine.fundingCode = invoiceLine.schemeCode
+    delete invoiceLine.schemeCode
```

Comment:
What's the value in deleting the `schemeCode` from the object?

If it doesn't exist in the database columns it would just be ignored by the Sequelize create query.